exercises/hello-world/HINT.md

```diff
@@ -0,0 +1,4 @@
+The starter implementation for this exercise uses [UnsupportedOperationException](http://docs.oracle.com/javase/8/docs/api/?java/lang/UnsupportedOperationException.html). 
```

Comment:
Is 'starter implementation' terminology that practitioners are familiar with? Maybe 'default' implementation would be easier to understand here if not?